src/app/app.module.ts

```diff
@@ -15,7 +21,7 @@ import { LineChartComponent } from './components/line-chart/line-chart.component
 import { LeftColumnComponent } from './components/left-column/left-column.component';
 import * as fromGraphRoute from './state/graph-route/graph-route.reducer';
 import { HttpTableComponent } from './components/http-table/http-table.component';
-import { TempFileUploadComponent } from './components/temp-file-upload/temp-file-upload.component';
+import StartLoadTestWidgetComponent from './components/start-load-test-widget/start-load-test-widget.component';
```

Comment:
When the component is refactored to not be a default export, this import will need curly braces added.